terraform/root_locals.tf

```diff
@@ -1,12 +1,12 @@
 locals {
   environment = lower(terraform.workspace)
-  assume_role = local.environment == "mgmt" || local.environment == "sbox" || local.environment == "ddri" ? "arn:aws:iam::${var.tdr_account_number}:role/IAM_Admin_Role" : "arn:aws:iam::${var.tdr_account_number}:role/TDRTerraformRole${title(local.environment)}"
+  assume_role = var.project == "dr2" ? "arn:aws:iam::${var.account_number}:role/${title(local.environment)}TerraformRole" : local.environment == "mgmt" || local.environment == "sbox" || local.environment == "ddri" ? "arn:aws:iam::${var.account_number}:role/IAM_Admin_Role" : "arn:aws:iam::${var.account_number}:role/TDRTerraformRole${title(local.environment)}"
```

Comment:
Referencing a specific TDR IAM role: `TDRTerraformRole ...` is that correct.

Reply:
There's a change here https://github.com/nationalarchives/da-terraform-configurations/pull/3 that adds the account role which I've used here.